src/client/js/main.js

```diff
@@ -0,0 +1,5 @@
+import { loadInjection } from './Injection.js'; // eslint-disable-line
+import inject from './Injection.js'; // eslint-disable-line
```

Comment:
Linter doesn't like this same file being imported twice, but compiler doesn't like importing defaults in {} so I just told the linter to f off.